README.md

```diff
@@ -2,13 +2,37 @@
 
 A microservice to securely transfer documents.
 
-## Launching
+## Running
 
-To launch the service, install dependencies using `bundle install`, then run the
-following:
+### Docker
```

Comment:
Maybe add a line here about making sure that you have Docker Desktop installed on your machine and running locally:
https://docs.docker.com/desktop/